app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
```

Comment:
U can use here only one import here, but u should use best practices for breaking a line in import

Reply:
from package_name import (
    x,
    y,
    z,
)

Reply:
Much obliged!